adapters/lib.js

```diff
@@ -41,6 +41,18 @@ async function isChannelValid(cfg, channel, address) {
 			'channel.depositAsset is not whitelisted'
 		)
 	}
+	// we split to remove space JSON.stringify adds
+	// between keys
+	const formattedSpec = JSON.stringify(channel.spec)
```

Comment:
don't do this, since it will remove the spaces in the strings too

just count it as it is...despite the spaces